components/ExercisesList/index.js

```diff
@@ -0,0 +1,21 @@
+import ExercisePreview from "../ExercisePreview";
+import styled from "styled-components";
+
+export default function ExercisesList({ exercises }) {
+  return (
+    <>
+      <List>
+        {exercises.map((exercise) => (
+          <ExercisePreview key={exercise.id} exercise={exercise} />
+        ))}
+      </List>
+    </>
```

Comment:
```suggestion
      <List>
        {exercises.map((exercise) => (
          <ExercisePreview key={exercise.id} exercise={exercise} />
        ))}
      </List>
```
for one parent element in the return you don't need the fragments :)